tests/CrudPanel/CrudPanelColumnsTest.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace CrudPanel;
+
+class CrudPanelColumnsTest extends BaseCrudPanelTest
+{
+    /** @test */
+    public function it_can_add_column_by_name()
+    {
+        $expectedColumn = [
+            'name'  => 'column',
+            'label' => 'Column',
+        ];
+
+        $this->crudPanel->addColumn('column');
+
+        $this->assertEquals(1, count($this->crudPanel->columns));
+        $this->assertContains($expectedColumn, $this->crudPanel->columns);
+    }
+
+    /** @test */
+    public function it_can_add_column()
+    {
+        $expectedColumn = [
+            'name'  => 'column',
+            'label' => 'Column',
+        ];
+
+        $this->crudPanel->addColumn($expectedColumn);
+
+        $this->assertEquals(1, count($this->crudPanel->columns));
+        $this->assertContains($expectedColumn, $this->crudPanel->columns);
+    }
+
+    /** @test */
+    public function it_can_add_columns_by_name()
+    {
+        $expectedColumns = [
+            [
+                'name'  => 'column',
+                'label' => 'Column',
+            ],
+            [
+                'name' => 'column2',
+                'label' => 'Column2',
+            ],
+        ];
+
+        $this->crudPanel->addColumns(['column', 'column2']);
+
+        $this->assertEquals(2, count($this->crudPanel->columns));
+        $this->assertContains($expectedColumns[0], $this->crudPanel->columns);
+        $this->assertContains($expectedColumns[1], $this->crudPanel->columns);
+    }
+
+    /** @test */
+    public function it_can_add_columns()
+    {
+        $expectedColumns = [
+            [
+                'name'  => 'column',
+                'label' => 'Column',
+            ],
+            [
+                'name' => 'column2',
+                'label' => 'Column2',
+            ],
+        ];
+
+        $this->crudPanel->addColumns($expectedColumns);
```

Comment:
It might actually be "more valid" to change the "label" to something that we know they wouldn't be if just called by 'name' otherwise we really don't know whether the default labeller labelled it, the label attribute was used or we just got lucky.